### PR TITLE
[NEW][11] Tree view color line (like before V11)

### PR DESCRIPTION
**Pull request by fmdl**
Dear

In V11, the attribute `colors` in tree view is no more supported by Odoo (deprecated from V9). It replace by the decoration system. But if you would like to hard a new color, it is very hard (you need to modifiy python code, rng, javascript and css).

This module re-add the old fonctionality exemple : `colors="blue:tetris == 'blue';red:tetris == 'red'"`.

@pedrobaeza what do you think ?

### CODE REVIEW

File: web_tree_color/README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Web Tree Color
```
Comment (elicoidal) [inline web_tree_color/README.md]: You need to add a README.rst following https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst

File: web_tree_color/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline web_tree_color/__init__.py]: no coding line in v11

File: web_tree_color/__manifest__.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
```
Comment (elicoidal) [inline web_tree_color/__manifest__.py]: please add short headers

File: web_tree_color/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+{
+    'name': "Web Tree Color",
+    'summary': "Web Tree Color",
+    'author': "Florent de Labarre",
+    'category': 'Custom',
+    'website': 'https://github.com/fmdl',
+    'version': '1.0',
```
Comment (elicoidal) [inline web_tree_color/__manifest__.py]: 11.0.1.0.0

File: web_tree_color/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "Web Tree Color",
+    'summary': "Web Tree Color",
+    'author': "Florent de Labarre",
+    'category': 'Custom',
+    'website': 'https://github.com/fmdl',
```
Comment (elicoidal) [inline web_tree_color/__manifest__.py]: This repo

File: web_tree_color/static/src/js/list_view.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
+odoo.define("web_tree_color.ListView", function(require) {
+    "use strict";
+
```
Comment (elicoidal) [inline web_tree_color/static/src/js/list_view.js]: add copyright

File: web_tree_color/__manifest__.py [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+    'version': '1.0',
+    'depends': ['web'],
+    'data': ['views/web_tree_color.xml'],
+    'demo': [],
```
Comment (elicoidal) [inline web_tree_color/__manifest__.py]: please remove empty or default keys

**Review by elicoidal — CHANGES_REQUESTED** [review]
Thanks for the contribution

**Comment by fmdl** [comment]
Dear @elicoidal , thank for you comment.
My PR is not finish, I would like your approbation before make a clean module.

**Comment by elicoidal** [comment]
no problem: ping me when necessary

File: web_tree_color/static/src/js/list_view.js (hunk 6 above)
Comment (fmdl) [reply web_tree_color/static/src/js/list_view.js]: Do

**Comment by fmdl** [comment]
@elicoidal can you merge ?

**Comment by pedrobaeza** [comment]
There must be at least 2 reviewers. You can review other PRs and ask in exchange that they review yours.

**Comment by fmdl** [comment]
@alnslang can you review ?

File: web_tree_color/static/src/js/list_view.js (hunk 6 above)
Comment (fmdl) [reply web_tree_color/static/src/js/list_view.js]: @elicoidal ping
Thanks

File: web_tree_color/README.rst [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+-------
+
+    <tree string="Stock Move" colors="blue:tetris == 'blue';red:tetris == 'red'">
+        <field name="tetris"/>
+    </tree>
+
+Color codage
+------------
+
+* css standard name : red
```
Comment (elicoidal) [inline web_tree_color/README.rst]: remove space before ':'

File: web_tree_color/README.rst [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+* hexadecidmal : #CF403E
+
+
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/web/162/11.0
+
+
+Known issues / Roadmap
```
Comment (elicoidal) [inline web_tree_color/README.rst]: remove if empty

File: web_tree_color/README.rst [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+======================
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
```
Comment (elicoidal) [inline web_tree_color/README.rst]: please correct the URL

File: web_tree_color/README.rst [hunk 11]
```diff
@@ -0,0 +1,75 @@
...
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://odoo-community.org/logo.png>`_.
+
+Contributors
+------------
+
+* Florent de Labarre
```
Comment (elicoidal) [inline web_tree_color/README.rst]: can you add a mail?

File: web_tree_color/__manifest__.py [hunk 12]
```diff
@@ -0,0 +1,15 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': "Web Tree Color",
+    'summary': "Web Tree Color",
```
Comment (elicoidal) [inline web_tree_color/__manifest__.py]: a short description is appreciated

File: web_tree_color/__manifest__.py [hunk 13]
```diff
@@ -0,0 +1,15 @@
...
+    'author': "Odoo Community Association (OCA), Florent de Labarre",
+    'category': 'Web',
+    'license': 'AGPL-3',
+    'website': 'https://github.com/OCA/web',
+    'version': '11.0.1.0.0',
+    'depends': ['web'],
+    'data': ['views/web_tree_color.xml'],
+    'installable': True,
+    'auto_install': False,
```
Comment (elicoidal) [inline web_tree_color/__manifest__.py]: are the following 2 really needed?

File: web_tree_color/__manifest__.py [hunk 14]
```diff
@@ -0,0 +1,15 @@
...
+{
+    'name': "Web Tree Color",
+    'summary': "Web Tree Color",
+    'author': "Odoo Community Association (OCA), Florent de Labarre",
+    'category': 'Web',
+    'license': 'AGPL-3',
+    'website': 'https://github.com/OCA/web',
+    'version': '11.0.1.0.0',
+    'depends': ['web'],
+    'data': ['views/web_tree_color.xml'],
```
Comment (elicoidal) [inline web_tree_color/__manifest__.py]: do you have a way to add a simple demo data?
(nice to have)

**Review by elicoidal — CHANGES_REQUESTED** [review]
small details. 
demo data are nice to have
LGTM (no test: but can I without demo data nor UT)

**Comment by fmdl** [comment]
@elicoidal runbot seems doesn't work, can you check ?

**Comment by elicoidal** [comment]
I can access runbot but it is far too slow from my location (everypage needs 1 minute to load)

**Review by elicoidal — COMMENTED** [review]
I do not know how to test the feature: a real example with screenshot in the README would be nice to have.

File: web_tree_dynamic_colored_line/demo/res_partner.xml [hunk 15]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="view_partner_tree" model="ir.ui.view">
```
Comment (fmdl) [inline web_tree_dynamic_colored_line/demo/res_partner.xml:3]: @elicoidal I have add an demo file.

Comment (elicoidal) [reply web_tree_dynamic_colored_line/demo/res_partner.xml:3]: Awesome!
I tried Runbot but it is empty (no menu at all)

**Pull request closed without merging**